### ISSUE TITLE
grid height calculations

### ISSUE THREAD

**Issue by ShadesOB** [jimrhoskins/ngmodules.org#18]
This isn't a bug per say. It's just something I had to re-write in my version of ng-grid.
What's the go with the mass amount of code you guys used to calculate the height of everything?  Is it so that it works on phones or something?

Anyway, I just finished modifying everything so that all the height settings are optional.
As an example:  Now if you don't specify a rowHeight each row height it determined by the html in the row. The height of the grid is determined by the height of the rows.
So basically the height is now auto.

Also lol at all the divide by zero stuff.  Set rowHeight = 0  on official version and see what happens.

took about 30min. I recommend you guys do it for the official version.
